packages/vaadin-lumo-styles/src/components/rich-text-editor.css

```diff
@@ -5,60 +5,35 @@
  */
 @media lumo_components_rich-text-editor {
   :host {
-    display: flex;
-    flex-direction: column;
-    box-sizing: border-box;
     min-height: calc(var(--lumo-size-m) * 8);
     font-family: var(--lumo-font-family);
     font-size: var(--lumo-font-size-m);
     line-height: var(--lumo-line-height-m);
+    background: transparent;
+    border: none;
+    color: inherit;
```

Comment:
This property doesn't seem to be defined in the base styles. Is there a reason we need to set it to `inherit`?